e2e/node/e2e.test.ts

```diff
@@ -130,33 +141,58 @@ describe.each(serversUnderTest)(
   "VC client end-to-end tests authenticated to [%s], issuing from [%s] for [%s]",
   (
     oidcIssuerDisplay,
-    vcIssuerDisplay,
+    vcServiceDisplay,
     vcSubjectDisplay,
     clientId,
     clientSecret
   ) => {
     const oidcIssuer = "https://" + oidcIssuerDisplay;
-    const vcIssuer = "https://" + vcIssuerDisplay;
+    const vcService = "https://" + vcServiceDisplay;
     const vcSubject = "https://" + vcSubjectDisplay;
 
-    describe("issueVerifiableCredential", () => {
-      it("has the appropriate environment variables", () => {
-        expect(oidcIssuer).not.toBeUndefined();
-        expect(clientId).not.toBeUndefined();
-        expect(clientSecret).not.toBeUndefined();
-        expect(vcIssuer).not.toBeUndefined();
-        expect(vcSubject).not.toBeUndefined();
+    it("has the appropriate environment variables", () => {
+      expect(oidcIssuer).not.toBeUndefined();
+      expect(clientId).not.toBeUndefined();
+      expect(clientSecret).not.toBeUndefined();
+      expect(vcService).not.toBeUndefined();
+      expect(vcSubject).not.toBeUndefined();
+    });
+
+    const session = new Session();
+    // let vcConfiguration: Partial<{
+    //   derivationService: string;
+    //   issuerService: string;
+    //   statusService: string;
+    //   verifierService: string;
+    // }>;
+
+    beforeEach(async () => {
+      await session.login({
+        oidcIssuer,
+        clientId,
+        clientSecret,
       });
+      // The following code snippet doesn't work in Jest, probably because of
+      // https://github.com/standard-things/esm/issues/706 which seems to be
+      // related to https://github.com/facebook/jest/issues/9430. The JSON-LD
+      // module depends on @digitalbazaar/http-client, which uses esm, which
+      // looks like it confuses Jest. Working around this by hard-coding the
+      // endpoints IRIs.
```

Comment:
Can we fork or patch JSON-LD to just use `fetch()`?

Reply:
That said, it looks like ESM is coming to jsonld: https://github.com/digitalbazaar/jsonld.js/issues/399

Reply:
I don't think forking or patching should be necessary, if I understand correctly `json-ld.js` supports a custom document loader being provided. I created a ticket to that intent.